app/league/slack_messenger.py

```diff
@@ -28,6 +26,9 @@ def init_app(self, app):
 
     def update_configuration(self, config):
         """Update Slack Messenger configuration."""
+        assert 'enabled' in config
+        if type(config['enabled']) == str:
```

Comment:
It is true that structured data lends itself better to something like Mongo, but performance is not really gonna be an issue here